src/profile_manager/views.py

```diff
@@ -157,6 +176,35 @@ def get_object(self):
         return self.request.user.profile
 
 
+@method_decorator(staff_member_required, name='dispatch')
+class PasswordReset(FormView):
+    form_class = SetPasswordForm
+    template_name = 'profile_manager/profile_form.html'
```

Comment:
Why is this using the profile form template? Seems like this should have its own template, though maybe it would be identical?  Actually is "profile_form" tempalte even used anywhere?  if it's not, rename to something more explicit and obvious, like password_change_form.html

Reply:
im using the profile_form template since it looks super generic and could work with almost any form
![image](https://user-images.githubusercontent.com/39788517/177684914-1d2fdcfc-03dc-4bb4-a402-ffd84cda97e1.png)
 will change name to password_change_form.html